src/app/app.component.ts

```diff
@@ -15,30 +16,31 @@ export class AppComponent implements OnInit {
   countryList = countryList;
   filteredCountries = countryList;
   searchContent: string = '';
-  activeUrlIndex: Number;
+  activeUrlIndex: number;
 
   isAuthenticated() {
     this.spotifyService.isAuthenticated();
   }
 
   constructor(
     private spotifyService: SpotifyService,
-    private router: Router
+    private router: Router,
+    private browserStorageService: BrowserStorageService
   ){
     this.isCollapsed = true;
     this.setActivePage(this.router.url);
   }
 
   ngOnInit(){
     this.onPageChange();
-    const code = localStorage.getItem('country') || 'FR';
-    const selectedCountry = countryList.find(country => country.code === code);
+    const countryCode = this.browserStorageService.getCountryCode('country');
```

Comment:
the last edit and it will be perfect:
this.browserStorageService.getCountryCode()
No need to. add "country" parameters, because. we call getCountry 👍

Reply:
Made changes. My bad.